frontstage/views/register/create_account.py

```diff
@@ -18,6 +18,8 @@
 def register():
     cryptographer = Cryptographer()
     form = EnrolmentCodeForm(request.form)
+    if form.enrolment_code.data:
+        form.enrolment_code.data = form.enrolment_code.data.strip()
```

Comment:
Would you explain why we're checking for form enrolment_code data then assigning new data to the the thing we were checking against?

Reply:
just stripping leading/trailing whitespaces

Reply:
^ And we check if it exists because you can't perform the `.strip()` on `None`